js/site-search.js

```diff
@@ -11,9 +11,13 @@ bcpl.siteSearch = (($, window, constants) => {
 
 	const onSearchTabClick = (clickEvent) => {
 		const $searchBtn = $(clickEvent.currentTarget);
-		$searchBtn
+		const buttonCaption = $searchBtn
```

Comment:
I think it might make sense to break the class stuff away from the buttonCaption varaible, but i have nothing concrete on this recommendation.

Reply:
I broken it into two operations.